service/create/keypair.go

```diff
@@ -0,0 +1,44 @@
+package create
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+type keyPairInput struct {
+	ec2Client   *ec2.EC2
+	clusterName string
+}
+
+func (s *Service) publicKeyMaterial() ([]byte, error) {
+	return ioutil.ReadFile(s.pubKeyFile)
```

Comment:
This is now bound to the underlying host FS. We should abstract this. The code will be better maintainable and testable when we do not have such runtime dependencies.